### ISSUE TITLE
Crash running the latest version

### ISSUE THREAD

**Issue by bzzrd** [Typhonragewind/meshcentral-docker#12]
Hi, 
First thanks a lot for your work.
I have an issue with the latest version crashing at startup:

```
meshcentral_1  | node[9]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.
meshcentral_1  |  1: 0x7f34cb8553cc node::Abort() [/lib/x86_64-linux-gnu/libnode.so.72]
meshcentral_1  |  2: 0x7f34cb85545b  [/lib/x86_64-linux-gnu/libnode.so.72]
meshcentral_1  |  3: 0x7f34cb8d7de2 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/lib/x86_64-linux-gnu/libnode.so.72]
meshcentral_1  |  4: 0x7f34cb8d7f16 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/lib/x86_64-linux-gnu/libnode.so.72]
meshcentral_1  |  5: 0x7f34cb8202c8 node::InitializeOncePerProcess(int, char**) [/lib/x86_64-linux-gnu/libnode.so.72]
meshcentral_1  |  6: 0x7f34cb8204ac node::Start(int, char**) [/lib/x86_64-linux-gnu/libnode.so.72]
meshcentral_1  |  7: 0x7f34cad8ed90  [/lib/x86_64-linux-gnu/libc.so.6]
meshcentral_1  |  8: 0x7f34cad8ee40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
meshcentral_1  |  9: 0x55780572c0f5 _start [node]
meshcentral_1  | /opt/meshcentral/startup.sh: line 28:     9 Aborted                 (core dumped) node node_modules/meshcentral
```
i'm running the docker-hub image with the non-mongodb docker-compose from the readme.
it's behind traefik as a reverse proxy handling tls and pointing at port 80 of the container

os: Alpine 3.15.4
Docker version 20.10.7, build f0df35096d5f5e6b559b42c7fde6c65a2909f7c5
docker-compose version 1.29.2, build unknown

It was auto updated via watchtower.

Let me know if I can provide any more information.

Best regards

**Comment by Typhonragewind** [comment]
Hey, thanks for letting me  know!
You're having the same problem as in here #11 . Trouble is, I can't replicate it on my machine.

Can you please try to start it with fresh settings? No need to delete yours, just point the persistence location in the docker-compose to another place)

**Comment by bzzrd** [comment]
Yes it's the same issue but I didn't want to hijack some else's thread.
I've started with a fresh directory and have the same issue.

**Comment by Typhonragewind** [comment]
Quick question, are you using it behind a reverse proxy ?

**Comment by bzzrd** [comment]
Yes, traefik v2

**Comment by Typhonragewind** [comment]
Oh, doh, i read but didn't register. Would you mind trying a fresh install with the REVERSE_PROXY variable set to false, please?

**Comment by bzzrd** [comment]
Same behavior

**Comment by Typhonragewind** [comment]
I'm getting a bit stumped. Can you please check your config file, to see if it is correct? The problem either lies in the json being malformed or some obscure bug upon building the docker image

**Comment by bzzrd** [comment]
I've checked the JSON and everything is fine with it, 
It seems logical as the config was working previously.

I've pulled the repo and used the container on my laptop, using arch and docker 20.10.14
it's pulling and running properly.

I've tried to upgrade docker to 20.10.14, removed and pulled back the image.
It's now working.

So I guess the fix is to upgrade docker to the latest version. According to what I see it can be related to ubuntu upgrading to 22.04. 

let me know if I can be of any further help.
Cheers.
